ca.desrt.dconf-editor.json

```diff
@@ -12,6 +12,7 @@
         "--filesystem=xdg-run/dconf",
         "--filesystem=host:ro",
         "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=GIO_EXTRA_MODULES=/app/lib/gio/modules/",
```

Comment:
I don't understand. This is the default path already? What does this solve?

Reply:
First off, thanks for the quick answer. Also, this might not be The Right Way of fixing, but it worked for me. Very open for "better" solutions. 

# Here's what I was experiencing:

## Running the flatpak'd dconf-editor without modifications
* `flatpak run ca.desrt.dconf-editor` 
* looking at custom-keybindings:
![Bildschirmfoto von 2021-06-23 08-30-33](https://user-images.githubusercontent.com/80450/123047441-954d7480-d3fd-11eb-959e-b18a23a0a9f8.png)

## Running the flatpak'd dconf-editor with GIO_EXTRA_MODULES
* `flatpak run --command=sh ca.desrt.dconf-editor`
* Then, inside the sandbox: `GIO_EXTRA_MODULES=/app/lib/gio/modules/ start-dconf-editor`
![Bildschirmfoto von 2021-06-23 08-34-34](https://user-images.githubusercontent.com/80450/123047681-de052d80-d3fd-11eb-9085-22fb0b081a3d.png)

# Some background
A [tool of mine](https://github.com/bk138/gromit-mpx) relies on hotkeys and sets them via dconf when on Wayland. Upgrading the runtime to GNOME 3.38 broke this as dconf is not shipped in the runtime anymore. I took 4e6afc9f4b798455ac72cca50914d5a2a8e5039e as a blueprint for adding dconf back in, but the sandboxed tool was still not able to "get through" to the host dconf. Adding that GIO_EXTRA_MODULES setting fixed it.

Reply:
Interesting. I still want to understand why this changes something first.

GLib is built against the `/app` prefix so this really *should* be the default value.

Do you have the env var set on your host?

Reply:
Nope. `env | grep -i gio` yields nothing.